.github/workflows/_func.yaml

```diff
@@ -52,6 +52,10 @@ on:
         required: false
         type: boolean
         default: false
+      juju-channel:
```

Comment:
You did not used this anywhere. It's need to be passed to [actions-operator](https://github.com/charmed-kubernetes/actions-operator/blob/main/action.yaml#L47).

Reply:
It need to define here so I can pass `juju-channel` through?

I can't pass to `actions-operator` if we don't define it.

Reply:
Yes, you did it right, but you need to use it in step with actions/operator.

Reply:
add single line (nol 85):
```yaml
        with:
          provider: ${{ inputs.provider }}
          juju-channel: ${{ inputs.juju-channel }}
```